### PR TITLE
user + proof pages + support multiple proofs/user

### PR DESCRIPTION
**Pull request by lsankar4033**
TODO:
- [ ] inform existing anons to change twitter URLs
- [x] local set intersection computation

addresses https://github.com/personaelabs/cred/issues/6

### CODE REVIEW

File: packages/frontend/src/hooks/useGetSetIntersection.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+
+export const useGetSetIntersection = () => {
+  // NOTE: eventually this should use caching on the backend
```
Comment (DanTehrani) [inline packages/frontend/src/hooks/useGetSetIntersection.ts:4]: yeah the json files are large (the largest 15MB) so better to cache it in the future 👍

**Comment by DanTehrani** [comment]
lgtm!

proved in all sets in staging any anon set is 2.
Screenshot 2023-09-30 at 9 15 20

**Comment by DanTehrani** [comment]
lgtm! @lsankar4033 

Proved in all sets on staging any anon set is 2. not sure who's the other person but sounds about right! and the attributes cards look good!

Screenshot 2023-09-30 at 9 15 20

File: packages/frontend/src/hooks/useGetSetIntersection.ts (hunk 1 above)
Comment (lsankar4033) [reply packages/frontend/src/hooks/useGetSetIntersection.ts:4]: yep yep

**Pull request merged**